resources/css/style.css

```diff
@@ -405,11 +405,159 @@ footer p {
 }
 
 
+/*-----------------------------------------*/
+/* SECTION DASHBOARD */
+/*-----------------------------------------*/
+
+
+.modal-dialog-dashboard {
+	max-width: 95%;
+}
+
+.modal-content-dashboard {
+	width: 100%;
+}
+
+.close-dashboard {
+	position: relative;
+	left: 100%;
+	cursor: pointer;
+}
+.cart-icon {
+	position: fixed;
+	padding: 15px 20px 15px 15px;
+	margin-top: 70px;
+	color: #fff;
+	cursor: pointer;
+	background-color: #648B79;
+	z-index: 100000;
+	font-size: 200%;
```

Comment:
Why not use pixels?

Reply:
Well, after i did this course: https://www.codecademy.com/learn/learn-responsive-design, i took an advice from here and it said : "Pixels, however, are fixed, hard coded values. When a screen size changes (like switching from landscape to portrait view on a phone), elements sized with pixels can appear too small, overflow the screen, or become completely illegible.", so i try not to use pixels wherever i can. What is your opinion on this? :)